src/parser.mjs

```diff
@@ -14,6 +14,7 @@ function parse(text, opts) {
   const parser = new engine({
     parser: {
       extractDoc: true,
+      version: opts.phpVersion,
```

Comment:
This won't be needed once https://github.com/glayzzle/php-parser/pull/1150 has landed.

Reply:
I think the idea is that we always try to parse with the latest PHP version, and only use the prettier version setting to control our printing (i.e. don't print features that are unsupported in the configured version). Since https://github.com/glayzzle/php-parser/pull/1150 is now landed, I don't really see a reason to change this. I'd be very open to change update the default for our version setting to something more reasonable though! :+1: